submissions/Olumide Jokotade/HTML-CSS/Frere-wears-Log-In.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+    <head>
+            <style>
+                    body {background-color: whitesmoke; font-family: Arial, Helvetica, sans-serif;}
```

Comment:
Please use an external CSS file

Reply:
Please review HTML and CSS sections of the styleguide and make the necessary corrections